README.md

```diff
@@ -4,7 +4,7 @@ A package for simplifying the EDA of different data types!
 
 ## About
 
-**INSERT SUMMARY PARAGRAPH**
+`pyxplor` is a comprehensive Python package designed to automate and streamline the Exploratory Data Analysis (EDA) process. Tailored for various data types including numeric, categorical, binary, and time series data, `pyxplor` aims to enhance data interpretation through a suite of specialized plotting functions. This package seeks to reduce the complexity and time invested in initial data analysis, making it an essential tool for data scientists and analysts at all levels.
```

Comment:
Does capitalization matter? I think it does but I'm not too sure. Correct me if I'm wrong